src/components/SearchBar.js

```diff
@@ -42,10 +45,25 @@ class SearchBar extends Component {
 
     render() {
         return (
-            <form onSubmit={this.handleSubmit}>
-                <input type="text" value={this.state.value} onChange={this.handleChange} />
-                <input type="submit" value="Submit" />
-            </form>
+            <div>
+                <MuiThemeProvider>
+                    {/*TODO: Make AutoComplete component do its job! (Hint: lines 52 & 12!)*/}
+                    <AutoComplete 
```

Comment:
Yeah, totally! I was just setting up the static components for now, but I do recall you mentioning using bleve for that purpose, so I had that in mind as I was cheffing this code :)